### PR TITLE
Added uptime-kuma

### PR DESCRIPTION
**Pull request by 4rkal**
Following the standard that you're writing the config file, I have added uptime-kuma a simple service that monitors uptime. Feel free to edit anything

### CODE REVIEW

File: services/uptime-kuma/service.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    image: louislam/uptime-kuma
+    container_name: uptime-kuma
+    volumes:
+      uptime-kuma: /app/data
+    ports:
+      '3001/tcp': 3001
+    restart: always
+
+volumes:
+  uptime-kuma:
```
Comment (quentinguidee) [inline services/uptime-kuma/service.yml]: A docker method description in Vertex doesn't follow exactly a `docker-compose.yml` file description. For example, there is no `container_name` tag in vertex as the name is already handled automatically by Vertex. Same for the `restart` and `volumes`! Here is a modification that should work.

```suggestion
methods:
  docker:
    image: louislam/uptime-kuma
    volumes:
      data: /app/data
    ports:
      '3001/tcp': 3001
```

**Review by quentinguidee — CHANGES_REQUESTED** [review]
Hi! Thanks a lot for your contribution to Vertex! 🎉 

Everything looks good except some keys that Vertex handles automatically (see my comment below).

**Comment by 4rkal** [comment]
Should be fixed now, thanks for pointing that out

**Review by quentinguidee — APPROVED** [review]

**Comment by quentinguidee** [comment]
Thank you!!

**Pull request merged**